ui/lib/utils/i18n.ts

```diff
@@ -1,11 +1,11 @@
-import i18next from 'i18next'
-import { initReactI18next } from 'react-i18next'
-import LanguageDetector from 'i18next-browser-languagedetector'
-
-import dayjs from 'dayjs'
 import 'dayjs/locale/en'
 import 'dayjs/locale/zh-cn'
 
+import dayjs from 'dayjs'
```

Comment:
do we need to import dayjs first?

Reply:
not necessary, it is auto formatted by my vscode extension.